client/src/components/Common/Button/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, ConfigProvider } from 'antd';
+import 'antd/dist/antd.css';
+
+const ButtonComponent = ({
+  Content,
+  disabled,
+  loading,
+  size,
+  block,
+  onClick,
+}) => (
+  <ConfigProvider direction="rtl">
+    <Button
+      type="primary"
```

Comment:
Try to delete primary type, you already change the static style.

Reply:
I use the primary style to help me style the button.